.pre-commit-config.yaml

```diff
@@ -37,6 +37,8 @@ repos:
     rev: 23.10.1
     hooks:
     - id: black-jupyter
+      types_or: [python, pyi, jupyter]
+      types_or: [python, pyi, jupyter]
```

Comment:
```suggestion
```

Seems there's a duplicated key here, so removing that.

Reply:
Oops, was picking this out of other changes, and somehow it applied the patch twice.